core/app/components/pubs/PubEditor/PubEditor.tsx

```diff
@@ -130,7 +130,6 @@ export async function PubEditor(props: PubEditorProps) {
 			{
 				pubId: props.pubId,
 				communityId: props.communityId,
-				userId: user?.id,
```

Comment:
i would maybe add a comment here explaining why we're not passing the userId. or perhaps we should change the function to require passing an arg like `skipPermissionsCheck` so it's clear this isn't a mistake